src/plugins/saved_objects_management/public/plugin.ts

```diff
@@ -65,11 +67,13 @@ export interface SavedObjectsManagementPluginStart {
   actions: SavedObjectsManagementActionServiceStart;
   columns: SavedObjectsManagementColumnServiceStart;
   namespaces: SavedObjectsManagementNamespaceServiceStart;
+  uiActions: UiActionsStart;
```

Comment:
Same goes here. These are the services that savedObjectmanagement provides other plugins. I dont think this is needed.